src/components/modal_window/modalWindow.js

```diff
@@ -0,0 +1,22 @@
+export const modalWindow = () => {
```

Comment:
The component must accept a callback function. The call is supposed to display the value of the callback inside the modal.